wrappers/python/tests/TestIntegrators.py

```diff
@@ -0,0 +1,103 @@
+import unittest
+import tempfile
+from datetime import datetime, timedelta
+from simtk.openmm import *
+from simtk.openmm.app import *
+from simtk.unit import *
+import math, random
+
+class TestIntegrators(unittest.TestCase):
+    """Test Python Integrator classes"""
+
+    def testMTSIntegratorExplicit(self):
+        """Test the MTS integrator on an explicit solvent system"""
+        # Create a periodic solvated system with PME
+        pdb = PDBFile('systems/alanine-dipeptide-explicit.pdb')
+        ff = ForceField('amber99sbildn.xml', 'tip3p.xml')
+        system = ff.createSystem(pdb.topology, cutoffMethod=PME)
+
+        # Split forces into groups
+        for force in system.getForces():
+            if force.__class__.__name__ == 'NonbondedForce':
+                force.setForceGroup(1)
+                force.setReciprocalSpaceForceGroup(2)
+            else:
+                force.setForceGroup(0)
+
+        # Create an integrator
+        integrator = MTSIntegrator(4*femtoseconds, [(2,1), (1,2), (0,8)])
+
+        # Run a few steps of dynamics
+        context = Context(system, integrator)
+        context.setPositions(pdb.positions)
+        integrator.step(10)
+
+        # Ensure energy is well-behaved.
+        state = context.getState(getEnergy=True)
+        if not (state.getPotentialEnergy() / kilojoules_per_mole < 0.0):
+            raise Exception('Potential energy of alanine dipeptide system with MTS integrator is blowing up: %s' % str(state.getPotentialEnergy()))
+
+        # Clean up
+        del context, integrator
```

Comment:
This line is a noop.  It's the end of the method, so those variables are about to go out of scope anyway.